src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/MethodCodeNode.cs

```diff
@@ -10,9 +10,9 @@ namespace ILCompiler.DependencyAnalysis
 {
     internal class MethodCodeNode : ObjectNode, IMethodNode, INodeWithCodeInfo, INodeWithDebugInfo
     {
-        public static readonly string StartSectionName = ".managedcode$A";
-        public static readonly string SectionName = ".managedcode$I";
-        public static readonly string EndSectionName = ".managedcode$Z";
+        public static readonly ObjectNodeSection StartSectionName = new ObjectNodeSection(".managedcode$A", CustomSectionAttributes.Executable);
```

Comment:
Wouldn't it better to have some static functions in ObjectNodeSection that defines all the sections like it is done for the readonly, text, data, ...?